doc/tutorial_ti_cc3200.md

```diff
@@ -1,283 +1,3 @@
-    * Please leave only ```Simplelink WiFi CC31xx/CC32xx``` the selected and continue installation process
-
-### Run the Code Composer Studio™ Uniflash Software
-
-* Plug in your CC3200 device and make sure that the J15 Jumper is set to ON
-* From ```File``` select ```New Configuration``` and select
-    * Connection: ```CC3x Serial(UART) Interface```
-    * Board or Device: ```SimpleLink WiFi CC3100/CC3200```
-* On the left panel under the ```System Files``` please highlight the ```/sys/mcuimg.bin```file
-* From the right panel press the ```Browse``` button right next to the ```Url``` field
-* Pick the ```name_of_your_project.bin``` from your ```workspace_name/project_name/RELEASE/```
-* From the left panel highlight ```CC31xx/CC32xx Flash Setup and Control```
-* Press ```Program``` button
-* Set the J15 jumper to OFF and restart your device it should now run the test program
-
-## Congratulations!
-
-You did it! You now have a CC3200 board connected and communicating with Xively.
-
-You should be able to go back to your device page on Xively CPM and see that its status is now `Connected` and within the logs see its `Device connected` lifecycle log.
-
-<img src="https://cloud.githubusercontent.com/assets/1428256/19814034/91b3b296-9d0a-11e6-813b-9eb7ca499350.png" width="600">
-
-When you enter the Product Launcher you will be able to see the graphical representation of data received from your CC3200 device such as temperature and button states. You can also turn on and off CC3200's LEDs' using this web page interface.
-
-<img src="https://cloud.githubusercontent.com/assets/1899893/20430715/39b6de32-ad96-11e6-8945-79442b3f77a9.png" width="600">
-
-## What to do next?
-
-### How to use a more secure configuration of wolfSSL with OCSP Stapling?
-
-For advanced security with OCSP support please take a look at the next tutorial: [CC3200 Xively tutorial Stage 2](./tutorial_ti_cc3200_advanced.md)
-
-### More coming soon
-
-_For now [please visit our docs](http://developer.xively.com/docs) and view some other guides and please let us know any feedback or questions that you may have regarding this guide or Xively in general!_
-
-## Common pitfalls or errors
-
-_More coming soon_
-
-##### Q. When I build the example application I get the "Xively Hello World" debug message, but with a state of 34.
-
-    connection to broker.xively.com:8883 has failed reason 34
-
-**A.** A state of `34` means that the device connected to the Xively system, but its credentials are invalid. This could occur if you copied the credentials incorrectly or if you have regenerated the device credentials and are using older ones. The easiest way to fix this issue is to regenerate the device credentials (see Step 5.3) and re-copy the new credentials within `main.c`. Once you've done this rebuild the image flash the hardware again.
+This document has been moved and is accessible on [developer.xively.com](https://developer.xively.com/docs/ti-cc3200)
```

Comment:
Thanks for nit-picking :) Always welcome! :) So... I was thinking maybe too much ahead and I thought that since we had two files in our repository - there might have been created some external links ( most probably they haven't been created ) - it's always better to point to something than to non-existing file. So that was my only reasonable argument yet not that reasonable :)
Let me know what do you think about that.